cmake/add_project_darknet.cmake

```diff
@@ -17,6 +17,9 @@ endif()
 
 if( VIAME_ENABLE_CUDA )
   FormatPassdowns( "CUDA" VIAME_CUDA_FLAGS )
+  set( VIAME_DARKNET_CUDA_FLAGS 
+    -DCUDA_TOOLKIT_ROOT_DIR:PATH=${CUDA_TOOLKIT_ROOT_DIR}
+    -DCMAKE_LIBRARY_PATH:PATH=${CUDA_TOOLKIT_ROOT_DIR}/lib64/stubs )
```

Comment:
Should this be:

-DCMAKE_LIBRARY_PATH:PATH=${CUDA_TOOLKIT_ROOT_DIR}/lib64/stubs:${CMAKE_LIBRARY_PATH}

Legit question, I don't know the answer.

Reply:
Also as opposed to making a second variable, appending them to VIAME_CUDA_FLAGs or just changing VIAME_CUDA_FLAGS to VIAME_DARKNET_CUDA_FLAGS probably makes sense to prevent 2 variables from doing the same thing

Reply:
> Should this be:
> 
> -DCMAKE_LIBRARY_PATH:PATH=${CUDA_TOOLKIT_ROOT_DIR}/lib64/stubs:${CMAKE_LIBRARY_PATH}
> 
> Legit question, I don't know the answer.

I don't think so, or at least I haven't seen needing to do that in any cmake code.

Reply:
> Also as opposed to making a second variable, appending them to VIAME_CUDA_FLAGs or just changing VIAME_CUDA_FLAGS to VIAME_DARKNET_CUDA_FLAGS probably makes sense to prevent 2 variables from doing the same thing

My initial thought was I didn't want to append to VIAME_CUDA_FLAGS incase it was called again later on for other projects. (VIAME_CUDA_FLAGS sounds very general)

Reply:
It is used in other projects, but it gets over-written here and in everyone so it's only used internally to the file (though we could also call both DARKNET_CUDA flags, I'll make a quick commit after merge)